WordPress/WordPressIntents/en.lproj/Sites.strings

```diff
@@ -1,12 +1,17 @@
+/* This text is used when the user is configuring the iOS widget and selecting which WordPress site they want the widget to be for */
 "BOl9KQ" = "There are ${count} options matching ‘${site}’.";
 
+/* This text is used when the user is configuring the iOS widget, as a label for the dropdown to select the site to configure it for */
 "ILcGmf" = "Site";
 
+/* This text is used when the user is configuring the iOS widget, as the title for the modal when selecting the site to configure it for */
 "cyajMn" = "Site";
 
+/* This text is used when the user is configuring the iOS widget */
 "gpCwrM" = "Select Site";
 
+/* This text is used when configuring the iOS widget and confirming the WordPress site the user wants the widget to be for */
 "s4dJhx" = "Just to confirm, you wanted ‘${site}’?";
 
+/* This text is used when the user is configuring the iOS widget */
 "tVvJ9c" = "Select Site Intent";
```

Comment:
I tried playing with the iOS Widget and couldn't find a place anywhere in the widget's configuration screen where this string is used, so I think it's totally unused after all?

Reply:
Confirmed with Diego in Slack: we agree that this is very likely not to be shown to end users at all.

In fact, even if it were, I doubt that the term "Intent" here would mean much to end users, considering that term comes from the technical name that Apple gives to those ways of configuring data to widgets (and Siri and such) via `.intentdefinition` files, a term that could speak to us devs but will not mean anything to end users.

For that reason, I've decided to remove that term from the `.strings` files (909570f) so that it gets removed from GlotPress (thus avoiding confusing polyglots not knowing how to translate it), and does not get translated for naught in the end.